.github/workflows/release.yml

```diff
@@ -0,0 +1,120 @@
+name: "Release New Version"
+run-name: "Release ${{ inputs.version }}"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The version to be released. This is checked for consistency with the branch name and configuration"
+        required: true
+        type: "string"
+      jira-version-number:
+        description: "JIRA version ID (e.g. 54321)"
+        required: true
+        type: "string"
+
+env:
+  # TODO: Use different token
+  GH_TOKEN: ${{ secrets.MERGE_UP_TOKEN }}
+  GIT_AUTHOR_NAME: "DBX PHP Release Bot"
+  GIT_AUTHOR_EMAIL: "dbx-php@mongodb.com"
+  default-release-message: |
+    The PHP team is happy to announce that version {0} of the MongoDB integration for Laravel is now available.
+
+    **Release Highlights**
+
+    TODO: one or more paragraphs describing important changes in this release
+
+    A complete list of resolved issues in this release may be found in [JIRA](https://jira.mongodb.org/secure/ReleaseNote.jspa?version={1}&projectId=22488).
```

Comment:
This seems like something to change given the [existing releases](https://github.com/mongodb/laravel-mongodb/releases). I'm not familiar with how "Release Highlights" are generated, but we definitely don't link to JIRA. We _do_ have a link for "Full Changelog", though.

Reply:
I'd prefer to use GitHub milestones for that.
The full git history is better for now.

Reply:
Re-added automatic release note generation and removed our own release notes template. We'll [customise release note generation](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuring-automatically-generated-release-notes) to exclude merge-up pull requests and categorise other pull requests.